packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -569,10 +569,11 @@ export default class New extends Component {
 
     let { name, description } = this.refs
 
-    name = trim(name)
-    description = trim(description)
+    name = trim(name.value)
```

Comment:
`trim()` ?

Reply:
What do you mean?

Reply:
`String.prototype.trim()`   
JS function ^_^

Reply:
While I agree that it would be better to use `String#trim`, I'm not sure it is the point of this PR :slightly_smiling_face:

Reply:
Ok, I agree but this PR only aims to fix a bug so I'll avoid unrelated changes.

Reply:
The timing ;)